controllers/comments.controller.js

```diff
@@ -20,5 +20,32 @@ function getCommentsByArticleId (req, res, next) {
 
 }
 
+//POST COMMENT TO ARTICLE
+function postCommentByArticleId (req, res, next) {
+    
+    const article_id = req.params.article_id;
+    const comment = req.body;
+    const { username, body } = comment;
```

Comment:
Rather than having these 2 lines, it must be worth just restructuring `username` and `body` straight off `req.body`, instead of creating a variable `comment` then destructuring off that.